package-overrides/github/formly-js/angular-formly@5.0.3.json

```diff
@@ -0,0 +1,6 @@
+{
+  "registry": "jspm",
+  "dependencies": {
+    "api-check": "^6.0.11"
```

Comment:
It's better to write this as "api-check": "github:kentcdodds/apiCheck.js@^6.0.11", and leave out the `registry` option, as the registry isn't guaranteed not to change at a later point in time.